pom.xml

```diff
@@ -99,5 +104,17 @@
                 <testng.suite>${testng.configs.path}/allTests.xml</testng.suite>
             </properties>
         </profile>
+        <profile>
+            <id>exerciseFirstTests</id>
+            <properties>
+                <testng.suite>${testng.configs.path}/exercisefirst.xml</testng.suite>
+            </properties>
+        </profile>
+        <profile>
+            <id>exerciseSecondTests</id>
+            <properties>
+                <testng.suite>${testng.configs.path}/exercisesecond.xml</testng.suite>
+            </properties>
+        </profile>
```

Comment:
Why do you need two profiles instead of one for the hw2?

Reply:
I thought that for each class should be own profile